server/integration_tests/test_data/detection_api_sdg/chart_config.json

```diff
@@ -13,17 +14,5 @@
     "country/USA"
   ],
   "sessionId": "007_999999999",
-  "variables": [
-    "dc/topic/sdg_3",
-    "dc/topic/sdg_3.8",
-    "dc/topic/sdg_3.8.1",
-    "dc/topic/sdg_3.d.1",
-    "dc/topic/sdg_3.c.1",
-    "dc/topic/sdg_3.b.3",
-    "dc/topic/sdg_3.d",
-    "dc/topic/sdg_3.4.1",
-    "dc/topic/sdg_3.c",
-    "dc/topic/sdg_3.4",
-    "dc/topic/sdg_3.b.2"
-  ]
+  "variables": []
 }
```

Comment:
why this diff?

Reply:
caused by updating goldens without clearing embeddings cache first